lib/line/bot/client.rb

```diff
@@ -145,6 +145,40 @@ def revoke_channel_access_token_jwt(access_token)
         post(oauth_endpoint, endpoint_path, payload, headers)
       end
 
+      # Verify ID token
+      #
+      # @param id_token [String] ID token
+      # @param nonce [String] Expected nonce value. Use the nonce value provided in the authorization request. Omit if the nonce value was not specified in the authorization request.
+      # @param user_id [String] Expected user ID.
+      #
+      # @return [Net::HTTPResponse]
+      def verify_id_token(id_token, nonce: nil, user_id: nil)
+        channel_id_required
+
+        endpoint_path = '/oauth2/v2.1/verify'
+        payload = URI.encode_www_form({
+          client_id: channel_id,
+          id_token: id_token,
+          nonce: nonce,
+          user_id: user_id
+        }.compact)
+        headers = { 'Content-Type' => 'application/x-www-form-urlencoded' }
+        post(oauth_endpoint, endpoint_path, payload, headers)
+      end
+
+      # Verify access token v2.1
+      #
+      # @param access_token [String] access token
+      #
+      # @return [Net::HTTPResponse]
+      def verify_access_token(access_token)
+        payload = URI.encode_www_form(
+          access_token: access_token
+        )
+        endpoint_path = "/oauth2/v2.1/verify?#{payload}"
```

Comment:
[nice to have]

I think it is simple to write the access token because it doesn't need to be encoded and passed on. Please let me know if I'm misunderstanding.

```suggestion
        endpoint_path = "/oauth2/v2.1/verify?access_token=#{access_token}"
```

Reply:
I'm not sure that the content of access token is free from escaping.
[This example response](https://developers.line.biz/en/reference/line-login/#issue-token-response) contains '/' in `access_token`, which should be encoded as '%2F'

Reply:
Thank you very much. I understand the specifications. I have no problem with the implementation as it is.